Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-3: No Person shall be a Senator who shall not have attained to the Age of thirty Years, and been nine Years a Citizen of the United States, and who shall not, when elected, be an Inhabitant of that State for which he shall be chosen.
+3: No Person shall be a Senator who shall not have attained to the Age of thirty five Years, and been twelve Years a Citizen of these United States, and who shall not, when selected, be an Inhabitant of that State for which he shall be chosen.
 
-4: The Vice President of the United States shall be President of the Senate, but shall have no Vote, unless they be equally divided.
+4: The Vice President of these United States shall be President of the Senate, but shall have no Vote, unless they be equally divided.
 
-5: The Senate shall chuse their other Officers, and also a President pro tempore, in the Absence of the Vice President, or when he shall exercise the Office of President of the United States.
+5: The Senate shall choose their other Officers, and also a President pro tempore, in the Absence of the Vice President, or when he shall exercise the Office of President of these United States.
 
-6: The Senate shall have the sole Power to try all Impeachments. When sitting for that Purpose, they shall be on Oath or Affirmation. When the President of the United States is tried, the Chief Justice shall preside: And no Person shall be convicted without the Concurrence of two thirds of the Members present.
+6: The Senate shall have the sole Power to try all Impeachments. When sitting for that Purpose, they shall be on Oath or Affirmation. When the President of these United States is tried, the Chief Justice shall preside: And no Person shall be convicted without the Concurrence of two thirds of the Members present.
 
-7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under the United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
+7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under these United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
 
 Section 4
-1: The Times, Places and Manner of holding Elections for Senators and Representatives, shall be prescribed in each State by the Legislature thereof; but the Congress may at any time by Law make or alter such Regulations, except as to the Places of chusing Senators.
+1: The Times, Places and Manner of holding Elections for Representatives, and for deciding the Appointments of Senators, shall be prescribed in each State by the Constitution thereof, but such Elections must occur every second Year from the date on which the State joins the Union.
```

Comment:
Note that if a new State were to join the Union in an off-election year, that State's elections would occur in the off year rather than the normal election year. This change is intentional; the Coattail effect on down-ballot races from Presidential races is a problem, because it allows one movement in one year to significantly change government policy. Movements should need to be sustained and see broad agreement before being able to enact broad policy changes.